scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/vfl/FLModel.scala

```diff
@@ -26,18 +26,25 @@ import org.apache.spark.sql.DataFrame
 class FLModel() {
   val estimator: VflNNEstimator = null
   def fit(trainData: DataFrame,
-          valData: DataFrame,
-          epoch : Int = 1) = {
+          epoch: Int = 1,
+          batchSize: Int = 4,
+          featureColumn: Array[String] = null,
+          labelColumn: Array[String] = null,
+          valData: DataFrame = null) = {
     val _trainData = DataFrameUtils.dataFrameToSample(trainData)
     val _valData = DataFrameUtils.dataFrameToSample(valData)
     estimator.train(epoch, _trainData.toLocal(), _valData.toLocal())
   }
-  def evaluate() = {
+  def evaluate(data: DataFrame = null,
+               batchSize: Int = 4,
+               featureColumn: Array[String] = null) = {
     estimator.getEvaluateResults().foreach{r =>
```

Comment:
Report error if `data` is not null

Reply:
Some of these not implemented has been done in #3687